CMakeLists.txt

```diff
@@ -0,0 +1,65 @@
+# require at least cmake v3.0.0, so we can use CTest
+cmake_minimum_required(VERSION 2.8...3.13)
+
+# compile flags: -Wall "enable all compiler warnings"
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wall /wd4514 /wd4668 /wd4062 /wd5267 /wd4061 /wd5264 /wd4820 /wd5038 /wd5045")
+# c++ language version level: c++11
+set(CMAKE_CXX_STANDARD 11)
+# cmake project name: "project1"
+project(libsat355 VERSION 0.1.0)
```

Comment:
comments: the project name isn't "project1".
also: switch the names of your project and build target:
```
    project <-- "libsat"
    lib target <-- "libsat355"
```